static/app/cloudfoundry.js

```diff
@@ -76,6 +76,23 @@
                 });
         };
 
+        // Get quota usage data
+        this.getQuotaUsage = function(org) {
+            $http.get('/v2/organizations/' + org.guid + '/memory_usage')
```

Comment:
@msecret great article! We have this kind of deep nesting in a few places. i really like this pattern because of the ease of error handling.